perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/FacilityAttributeProcessor.java

```diff
@@ -37,12 +39,23 @@ public void processAttributeSet(String msg, MessageBeans beans) {
 			log.debug("Getting resources assigned to facility {}", beans.getFacility().getId());
 			// List<Resource> resources = Rpc.FacilitiesManager.getAssignedResources(ldapcManager.getRpcCaller(), beans.getFacility());
 			List<Resource> resources = perun.getFacilitiesManager().getAssignedResources(ldapcManager.getPerunSession(), beans.getFacility());
+
```

Comment:
If you look at #2343, we will have new type of message for virtual attribute changes. I think we need to add new pattern (at the top of this file) and pass it to the ``super()`` in constructor. So it means you need to update ``AbstractAttributeProcessor`` to handle this type of message. Virtual attributes will be always "changed" and not set/removed.
So we can create new abstract method to handle this "changed" action without regards to the attribute type. User and FacilityAttributeProcessor will implement it.